### PR TITLE
Corrected link for trial-program

### PR DESCRIPTION
**Pull request by srivastava-jyoti**
Targets #{912}

Checklist:

- [x] Corrected link for Trial-Program
- [x] Removed blue color outline which appears on clicking tabs

### CODE REVIEW

File: resources/views/hr/application/index.blade.php [hunk 1]
```diff
@@ -59,16 +59,16 @@
             </li>
             <li id="list-styling">
                 <a id="job-application-listings" class="{{ $status === config('constants.hr.status.in-progress.label') ? 'job-application-status' : '' }} btn" 
-                href=/{{ Request::path() }}?status={{ config('constants.hr.status.in-progress.label') }}{{$query_filters}}>
-                    <sup class = "application-menu-options-title fz-20">
+                href=/{{ Request::path() }}?status={{ config('constants.hr.status.in-progress.label') }}{{$query_filters}}&round=Trial-Program>
+                    <sup class = "application-menu-options-title fz-20" {{ request()->get('round')=='Trial-Program'}} >
```
Comment (P4NK4J) [inline resources/views/hr/application/index.blade.php:63]: It seems that only conditional is being checked no further action is performed, can you elaborate on what is this doing ? Also, let's change the slug name to all small letters like `trail-program`, to maintain the consistency

**Review by P4NK4J — COMMENTED** [review]
left a comment, let me know your thoughts.

File: resources/views/hr/application/index.blade.php (hunk 1 above)
Comment (srivastava-jyoti) [reply resources/views/hr/application/index.blade.php:63]: Yeah, that request is just asking the user to move to the trial program not counting the no. applications in the trial program. So, it was not required. And I have also corrected the slug name all to lowecase.

**Review by rathorevaibhav — APPROVED** [review]

**Comment by rathorevaibhav** [comment]
I see feedbacks from @P4NK4J has been implemented. Merging this.

**Pull request merged**